scripts/generate_release_notes.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+
+from packaging.version import Version
+import json
+import os
+import re
+import requests
+import subprocess
+import sys
+
+# You can test this script locally with this command:
+# ALLOWED_LABELS="Breaking changes,New,Fixed,Improved,Changed,Removed,Deprecated" GITHUB_TOKEN="" GITHUB_REPO="Adyen/adyen-android" python ./scripts/generate_release_notes.py test_output.txt
+
+class DependencyUpdate:
+
+    def __init__(self, id, link, new_version, old_version):
+        self.id = id
+        self.link = link
+        self.new_version = new_version
+        self.old_version = old_version
+
+def fetch_latest_release_tag() -> str:
+    return subprocess.check_output(['git', 'describe', '--tags', '--abbrev=0']
+        ).decode(sys.stdout.encoding
+        ).strip()
+
+def fetch_recent_commits(latest_tag: str):
+    return subprocess.check_output(['git', 'log', '--oneline', latest_tag + '..HEAD']
+        ).decode(sys.stdout.encoding
+        ).strip()
+
+def is_merge_commit(commit: str) -> bool:
+    # Check if the commit message starts with a certain message after the hash
+    return commit.startswith('Merge pull request #', commit.index(' ') + 1)
+
+def get_pr_number(commit: str) -> str:
+    start = commit.index("#") + 1
+    end = commit.index(" ", start)
+    return commit[start:end]
+
+def fetch_pr(number: str) -> dict:
+    url = 'https://api.github.com/repos/{}/pulls/{}'.format(os.getenv('GITHUB_REPO'), number)
```

Comment:
What happens if we update a dependency without using renovate? Will this script skip it?